homework3/task02/project_fast_calculate/task2_fast_calculate.py

```diff
@@ -0,0 +1,27 @@
+"""
+Task.
+
+Here's a not very efficient calculation function that calculates something important.
+Calculate total sum of slow_calculate() of all numbers starting from 0 to 500.
+Calculation time should not take more than a minute. Use functional capabilities of multiprocessing module.
+You are not allowed to modify slow_calculate function.
+"""
+import hashlib
+import random
+import struct
+import time
+from multiprocessing import Pool
+
+
+def slow_calculate(value):
+    """Some weird voodoo magic calculations."""
+    time.sleep(random.randint(1, 3))
+    data = hashlib.md5(str(value).encode()).digest()
+    return sum(struct.unpack("<" + "B" * len(data), data))
+
+
+def fast_calculate():
+    """Return sum of slow_calculate() of all numbers starting from 0 to 500."""
+    with Pool(processes=60) as pool:
```

Comment:
Why 60?

Reply:
It is an optimal amount of processes that increase work of slow_calculate function for calculation time should not be more than a minute. 
For example. 100 processes are such a big amount and provide endless execution, lasting more than 60 seconds. 
20 processes work too slow, and in terms of 60 processes fast execution is reached.